pages/who-we-are/corporate/annual-report-15-16/01-introduction.md

```diff
@@ -0,0 +1,320 @@
+
+## Overview
+
+The DTO was established to assist in designing a more user-friendly government. We are creating unified digital platforms that will make it easier for all Australians to access the government services and information they need, and leading a user-focused, whole-of-government approach to digital transformation that will make public services simpler, clearer and faster for all.
+
+The role of the DTO is to design the system and governance for this change process in collaboration with our partners: it is not to own services or content. We facilitate collaboration and a whole-of-government approach while digital transformation takes place within the entities that provide government services and information.
+
+### Outcome and program structure
+
+The DTO has one portfolio outcome, delivered through one program:
+
+Outcome 1: To improve the user experience for all Australians accessing government information and services by leading the design, development and continual enhancement of whole-of-government service delivery policies and standards, platforms and joined-up services.
+
+Program 1.1: The Digital Transformation Office
+
+In 2015–16, the objective of Program 1.1 was:
+
+To support the Prime Minister in transforming government services to improve the user experience, cut red tape and ensure services are delivered efficiently.
+
+### Values
+
+At the heart of our values are the values of the Australian Public Service (APS) and a desire to transform government services to ensure that all users’ experiences are made simpler, clearer and faster. We encourage innovation by providing a nurturing and supportive environment.
+
+We are also committed to:
+
+   * devolving responsibility wherever possible, to enable improved decision making, accountability and communications
+   * acknowledging and valuing diversity and flexibility in the workplace
+   * providing opportunities for personal development and advancement
+   * promoting a culture of collaboration through robust and lively debate with internal and external stakeholders.
+
+### Legal framework
+
+The DTO was established as an executive agency under the *Public Service Act 1999* on 1 July 2015. It is a non-corporate Commonwealth entity subject to the *Public Governance, Performance and Accountability Act 2013* (PGPA Act).
+
+The DTO resides in the Prime Minister and Cabinet portfolio. The minister responsible for the DTO is the Prime Minister, the Hon Malcolm Turnbull MP, assisted by the Assistant Minister for Cities and Digital Transformation, the Hon Angus Taylor MP.
+
+The DTO does not administer any legislation.
+
+### Organisational structure
+
+  The structure of the agency at 30 June 2016 is shown in Figure 1.
+
+  ![Image of DTO Organisation Chart]({% asset_path org-chart %})
+  
+  DTO Organisational Chart
+  
+### Executive management
+
+<ul class="list-vertical">
+  <li>
+    <figure>
+    ![Image of Paul Shettler]({% asset_path DTO-Paul-130916-67 %})
```

Comment:
If you want to use markdown within a html block, you need to add the `markdown="1"` attribute to the surrounding block. https://kramdown.gettalong.org/syntax.html#html-blocks